client/js/components/procedure/basicSettings/ParticipationPhases.vue

```diff
@@ -0,0 +1,141 @@
+<template>
+  <div>
+    <div class="flex gap-2">
+      <div class="w-8-12">
+        <dp-select
+          :label="{
+            text: labelText,
+            tooltip: helpText
+          }"
+          :name="fieldName"
+          :options="phaseOptions"
+          required
+          v-model="selectedPhase" />
+      </div>
+
+      <div
+        v-if="true || hasPermission('field_phase_iterator')"
+        class="w-4/12 inline-block relative">
+        <dp-label
```

Comment:
Shouldn't there be an input with the id `r_public_participation_phase_iteration`?